docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+      }
+];
+
+await context.sync();
+```
+
+### [Preset criteria](https://docs.microsoft.com/javascript/api/excel/excel.presetcriteriaconditionalformat)
+
+Preset formatting applies a user-defined format to the range based on a selected standard rule. These rules are defined by the [ConditionalFormatPresetCriterion](https://docs.microsoft.com/javascript/api/excel/excel.ConditionalFormatPresetCriterion) in the [ConditionalPresetCriteriaRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionalpresetcriteriarule). The following example colors the font white wherever a cell’s value is at least one standard deviation above the range’s average.
+
+![A range with white font cells where the values are at least one standard deviation above average.](../images/excel-conditional-format-preset.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B2:M5");
+const conditionalFormat = range.conditionalFormats.add(
+     Excel.ConditionalFormatType.presetCriteria
+);
+
+// color every cell’s font white that is one standard deviation above average relative to the range
+conditionalFormat.preset.format.font.color = "white";
+conditionalFormat.preset.rule = {
+     criterion: Excel.ConditionalFormatPresetCriterion.oneStdDevAboveAverage
+};
+
+await context.sync();
+```
+
+### [Text comparison](https://docs.microsoft.com/javascript/api/excel/excel.textconditionalformat)
+
+Text comparison formatting uses string comparisons as the condition. The `rule` property is a [ConditionalTextComparisonRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionaltextcomparisonrule) defining a string to compare with the cell and an operator to specify the type of comparison. The following example formats the font color red when a cell’s text contains “Delayed”.
+
+![A range with cells containing "Delayed" in red.](../images/excel-conditional-format-text.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B16:D18");
+const conditionalFormat = range.conditionalFormats.add(
+     Excel.ConditionalFormatType.containsText
+);
+
+// color the font of every cell containing “Delayed”
+conditionalFormat.textComparison.format.font.color = "red";
+conditionalFormat.textComparison.rule = {
+     operator: Excel.ConditionalTextOperator.contains,
+     text: "Delayed"
+};
+
+await context.sync();
+```
+
+### [Top/bottom](https://docs.microsoft.com/javascript/api/excel/excel.TopBottomconditionalformat)
+
+Top/bottom formatting applies a format to the highest or lowest values in a range. Whether the condition is based on the highest or lowest, as well as whether the evaluation is ranked or percentage-based, is set by the `rule` property of type [ConditionalTopBottomRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionaltopbottomrule). The following example applies a green highlight to the highest value cell in the range.
```

Comment:
To be consistent with the opening sentence of other like paragraphs, consider starting this one with "Top/bottom conditional formatting..."